porthole/components.py

```diff
@@ -157,8 +157,9 @@ def create_record(self):
         self.report_log = report_log
         self.logger.extra.update(report_log_id=report_log.primary_key)
 
-    def finalize_record(self):
+    def finalize_record(self, all_recipients=None):
```

Comment:
Personally, I would call the variable `recipients` instead of `all_recipients` in the context of this particular method.

Reply:
I went with all_recipients to avoid ambiguity with to_recipients and cc_recipients in the reports module.  I kept the naming here because I liked the continuity of naming between the modules.  That being said, I don't feel strongly about it and am open to changing it.

Reply:
Seems fair to me. Thanks.